src/alternatingCase.js

```diff
@@ -0,0 +1,25 @@
+let toUpper = require("./toUpper");
+let toLower = require("./toLower");
+/**
+ * This method alternates the string its acalled with.
+ *
+ * author: charles-ebuka-chiakwa
+ */
+
+String.prototype.alternatingCase = function() {
```

Comment:
Give a space between function and () across your code.Read through this https://github.com/airbnb/javascript#iterators-and-generators.